### ISSUE TITLE
[BUG] require-hook and test.describe.configure

### ISSUE THREAD

**Issue by vladkrasn** [playwright-community/eslint-plugin-playwright#240]
Top-level `test.describe.configure` gets flagged with this rule enabled, don't think this shouldn't happen
![image](https://github.com/playwright-community/eslint-plugin-playwright/assets/28753668/826a20ab-345e-459a-97e7-e6a5f81678fb)

Also, I am not sure if it's by design, dotenv in `playwright.config.ts` is also getting flagged. Feels weird since it's not a test file and probably should be outside of scope of the plugin. Or is the expectation for this plugin is that it is only enabled for `.spec.` files?
![image](https://github.com/playwright-community/eslint-plugin-playwright/assets/28753668/19ea5e64-12ef-4ddf-b847-e20237d7f1a9)

**Comment by EvgenyOrekhov** [comment]
Another example [from the docs](https://playwright.dev/docs/api/class-test#test-describe-configure) that gets flagged:

```js
test.describe('A, runs in parallel with B', () => {
  test.describe.configure({ mode: 'default' });
  test('in order A1', async ({ page }) => {});
  test('in order A2', async ({ page }) => {});
});
```
